tests/test_feat_new_moments_based.py

```diff
@@ -0,0 +1,164 @@
+import pathlib
+
+import h5py
+import numpy as np
+
+from dcnum.feat import feat_moments
+
+data_path = pathlib.Path(__file__).parent / "data"
+
+
+def test_moments_based_features():
+    # This file has new cell features belonging to
+    # fmt-hdf5_cytoshot_full-features_2023.zip
+    file_name = ("new_feat_tests.hdf5")
+
+    feats = [
+            "area_um_raw",
+            "per_um_raw",
+            "deform_raw",
+            "eccentr_prnc",
+            "per_ratio",
+            "s_x",
+            "s_y",
+    ]
+
+    # Make data available
+    with h5py.File(data_path / file_name) as h5:
+        data = feat_moments.moments_based_features(
+            mask=h5["events/mask"][:],
+            pixel_size=0.2645
+        )
+        for feat in feats:
+            if feat.count("inert"):
+                rtol = 2e-5
+                atol = 1e-8
+            else:
+                rtol = 1e-5
+                atol = 1e-8
+            assert np.allclose(h5["events"][feat][:],
+                               data[feat],
+                               rtol=rtol,
+                               atol=atol), f"Feature {feat} mismatch!"
+
+
+def test_mask_0d():

```

Comment:
All tests that only check for `np.isnan` are redundant (in `test_feat_moments_based.py`) and can be removed here.